File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.java
import java.util.HashSet;
import java.util.List;
import java.util.ArrayList;
import java.util.Arrays;

class Solution {
    public int[] solution(int[] numbers) {
        int[] answer = {};
        HashSet<Integer> hs = new HashSet<>();
        
        for(int i = 0; i < numbers.length; i++) {
        	for(int j = 0; j < numbers.length; j++) {
                if(i != j) {
                    int sum = numbers[i] + numbers[j];
        		    hs.add(sum); 
                }

        	}
        }
        
        List<Integer> list = new ArrayList<>(hs);
        

        int[] arr = list.stream()
                .mapToInt(Integer::intValue)
                .toArray();
        
        answer = arr;
        
        Arrays.sort(arr);
        
        return answer;
    }
}

File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.py
def solution(numbers):
    answer = []

    seen = set()

    for i in range(0, len(numbers)):
        for j in range(i + 1, len(numbers)):
           num = numbers[i] + numbers[j]
           seen.add(num)

    answer = sorted(list(seen))

    return answer